### PR TITLE
Add new Bootstrap Icons font example

### PR DESCRIPTION
**Pull request by mdo**
Adds new example of using Bootstrap Icons with npm, Sass, and the icon font. Will follow-up with another that uses the SVGs. Curious if I can figure out a way to do a more performance SVG sprite, too.

### CODE REVIEW

File: icons-font/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "scripts": {
+    "build": "npm run css",
+    "css-compile": "sass --style compressed --source-map --embed-sources --no-error-css --load-path=node_modules scss/:css/",
+    "css-lint": "stylelint scss/",
+    "css-prefix": "postcss --replace css/styles.css --use autoprefixer --map",
+    "css": "npm-run-all css-compile css-prefix",
+    "server": "serve --listen 3000",
+    "start": "npm-run-all --parallel watch server",
+    "watch": "nodemon -e html,scss -x \"npm run css\"",
+    "test": "npm-run-all css-lint css"
```
Comment (julien-deramond) [inline icons-font/package.json:18]:
```sh
$ npm run test

> icons-font@0.0.0 test
> npm-run-all css-lint css


> icons-font@0.0.0 css-lint
> stylelint scss/

Error: No configuration provided for /home/ju/twbs/examples/icons-font/scss/styles.scss
    at module.exports (/home/ju/twbs/examples/icons-font/node_modules/stylelint/lib/utils/configurationError.js:11:49)
    at getConfigForFile (/home/ju/twbs/examples/icons-font/node_modules/stylelint/lib/getConfigForFile.js:65:4)
    at async isPathIgnored (/home/ju/twbs/examples/icons-font/node_modules/stylelint/lib/isPathIgnored.js:25:17)
    at async lintSource (/home/ju/twbs/examples/icons-font/node_modules/stylelint/lib/lintSource.js:37:20)
    at async /home/ju/twbs/examples/icons-font/node_modules/stylelint/lib/standalone.js:228:27
    at async Promise.all (index 0)
    at async standalone (/home/ju/twbs/examples/icons-font/node_modules/stylelint/lib/standalone.js:267:22)
ERROR: "css-lint" exited with 78.
```

File: icons-font/index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Bootstrap Sass &amp; JS</title>
```
Comment (julien-deramond) [inline icons-font/index.html:7]: Probably "Bootstrap Icons Font" or something like that.

**Review by julien-deramond — COMMENTED** [review]
LGTM :rocket: Just few details in comments.

**Comment by julien-deramond** [comment]
More general feedback. This is a generic example so that users can play with the icons, but IRL we'd purify the CSS to get rid of the unused icons.

**Comment by mdo** [comment]
Fixed the issues you mentioned and added PurgeCSS. Agreed on an SVG example, that'll come separately, ideally with a custom sprite.

**Review by julien-deramond — APPROVED** [review]

**Pull request merged**